wicker/core/datasets.py

```diff
@@ -68,15 +75,20 @@ def __init__(
         self._storage = storage
         self._treat_objects_as_bytes = treat_objects_as_bytes
 
-        self._column_bytes_file_cache: Optional[ColumnBytesFileCache] = None
+        self._column_bytes_file_reader: ColumnBytesFileReader = ColumnBytesFileReader(
```

Comment:
It would be nice to create this in the else condition of `if local_cache_path_prefix is not None:` rather than creating and then possibly re-creating `self._column_bytes_file_reader`.

Reply:
Done